src/styles/block/pruduct-card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  background-color: $white-color;
+  border: 1px solid #f3f3f3;
+  padding: 0 16px;
+
+  &__image {
+    justify-content: center;
+    display: flex;
+    height: 134px;
+    margin-top: 32px;
+  }
+
+  &__image-content {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__info {
+    margin-top: 40px;
+    margin-bottom: 0;
+    padding: 0;
+    color: $primary-color;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+  }
+
+  &__code {
+    display: flex;
+    margin-top: 4px;
+    height: 14px;
+    align-items: center;
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: left;
+    font-weight: inherit;
+  }
+
+  &__price {
+    display: flex;
+    margin-top: 24px;
+    justify-content: space-between;
+
+    &-text {
+      color: $secondary-color;
+      font-size: 12px;
+      font-weight: inherit;
+      line-height: 18px;
+      text-align: left;
+    }
+
+    &-value {
+      color: $primary-color;
+      font-size: 16px;
+      font-weight: $font-weight-value-button;
+      line-height: 18px;
+      text-align: right;
+    }
+  }
+
+  &__button {
+    display: flex;
+    box-sizing: border-box;
+    align-items: center;
+    justify-content: center;
+    height: 40px;
+    margin-block: 16px;
+    border-radius: 5px;
+    background-color: $blue-color;
+
+    &-buy {
+      background-color: inherit;
+      border: none;
+      text-transform: uppercase;
+      color: $white-color;
+      font-size: 14px;
+      font-weight: $font-weight-value-button;
+      line-height: 16px;
+      border-radius: 5px;
+      width: 100%;
+      height: 100%;
+      text-align: center;
+
+      &:hover {
+        color: $blue-color;
+      }
+    }
+
+    &--hover:hover {
+      background-color: $white-color;
+      border: 1px solid $blue-color;
+    }
+  }
+
+  &:hover {
+    transition-duration: 300ms;
+    transform: scale(1.2);
```

Comment:
The `transform` property should be used with caution as it can cause layout shifts. Ensure that the scaling effect does not disrupt the layout of surrounding elements.